brands/healthcare/clinic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "healthcare/clinic|ZoomCare": {
+    "countryCodes": ["us"],
+    "tags": {
+      "amenity": "clinic",
+      "brand": "ZoomCare",
+      "brand:wikidata": "Q64120374",
+      "brand:wikipedia": "en:ZoomCare",
+      "healthcare": "clinic",
```

Comment:
`urgent_care=yes` `healthcare:speciality=emergency` seems like an appropriate way to differentiate this kind of clinic from retail clinic and community health centers without resorting to `emergency=yes` (which is for EMS admission).

Reply:
I think it's better to reserve healthcare:speciality=emergency for actual emergency rooms, some urgent care facilities aren't doing much more than primary care.

Glancing at their website, it appears some of their locations do provide a high level of service, but not all of them, and they still say to go to the ER if it is a real emergency.

Reply:
So Zoom Care isn't just an urgent care sort of clinic. You can just use them as a normal doctor if you want. urgent_care=yes might make sense, but it's def not emergency specialty. It's people with colds and also just people getting physicals and yearly checkups.

Reply:
Source: ZoomCare is my go to when I'm sick and don't want to head across town to my regular doctor.

Reply:
urgent_care=yes is meant to indicate that walk-in type service is available, not that it is the only type of service available. I think I'm the main user of the tag (but I'm bad at documenting things on the wiki or asking tagging if they want to argue about things). https://overpass-turbo.eu/s/Jt5

Reply:
I meant above that `urgent_care=yes` would be appropriate here since that is a big part of what they offer.